java/impl/aws/src/main/java/co/worklytics/psoxy/aws/request/APIGatewayV2HTTPEventRequestAdapter.java

```diff
@@ -2,9 +2,11 @@
 
 import co.worklytics.psoxy.gateway.HttpEventRequest;
 import com.amazonaws.services.lambda.runtime.events.APIGatewayV2HTTPEvent;
+import com.google.api.client.util.IOUtils;
```

Comment:
doesn't see this used ... and doesn't seem like the kind of thing we should depend on from an API client library.  I'd likely prefer using apache commons equivalent